packages/testlab/src/disable-unused-local-error.ts

```diff
@@ -0,0 +1,21 @@
+// Copyright IBM Corp. 2018. All Rights Reserved.
+// Node module: @loopback/testlab
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+// tslint:disable-next-line:no-any
+let ref: any = true;
+
+// Disable compiler error: 'ref' is declared but its value is never read.
+ref = !ref;
+
+/**
+ * A helper function to create a dummy reference to a local value that's
+ * otherwise not used anywhere in the code. Use it to surpress "tsc" warning
+ * "(something) is declared but never used".
+ *
+ * @param value A value to mark as used for the compiler.
+ */
+export function disableUnusedLocalError<T>(value: T) {
```

Comment:
Do we expect `disableUnusedLocalError` to be used in tests only? In most cases, `@loopback/testlab` is a dev dependency. What if we have some code in `src` that want to use this hack?

Reply:
> Do we expect `disableUnusedLocalError` to be used in tests only? In most cases, `@loopback/testlab` is a dev dependency. What if we have some code in `src` that want to use this hack?

Good point. I _hope_ we won't have unused dependences in production code - that's why we are enabling this linting/compiler rule in the first place. So far, the production code does not need this hack after `tsc` fixed re-exports of implicit dependencies in `tsc@3.0.0`.

Having said that, if there comes a time that we need to keep an unused local in our code, then we will indeed need something like `disableUnusedLocalError` to be available in production code too.